img/svg-icons/print-icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
+    <path fill="#0592AF"  d="M15.833 6.667H4.167c-1.384 0-2.5 1.116-2.5 2.5v5H5V17.5h10v-3.333h3.333v-5c0-1.384-1.116-2.5-2.5-2.5zm-2.5 9.166H6.667v-4.166h6.666v4.166zm2.5-5.833c-.458 0-.833-.375-.833-.833 0-.459.375-.834.833-.834.459 0 .834.375.834.834 0 .458-.375.833-.834.833zM15 2.5H5v3.333h10V2.5z"/>
```

Comment:
Just wondering: do we want to change the fill colors to use the color variables?

Reply:
I was thinking of leaving the fill colors in many of the SVG definitions to prevent adding extra CSS.  We can easily remove it if we need to use the icon with multiple colors later on.  Keeps things minimal for now.